specification/columns/contractedcost.md

```diff
@@ -2,6 +2,8 @@
 
 Contracted Cost represents the cost calculated by multiplying [*contracted unit price*](#glossary:contracted-unit-price) and the corresponding [Pricing Quantity](#pricingquantity). Contracted Cost is denominated in the [Billing Currency](#billingcurrency) and is commonly used for calculating savings based on negotiation activities, by comparing it with [List Cost](#listcost). If negotiated discounts are not applicable, the Contracted Cost defaults to the List Cost.
 
+When aggregating Contracted Cost for savings calculations, it's important to exclude either one-time or recurring charges ([ChargeCategory](#chargecategory)"Purchase") that are paid to cover future eligible charges or the covered charges themselves. This exclusion helps prevent double counting of these charges in the aggregation. Which set of charges to exclude depends on whether cost are aggregated on a billed basis (exclude covered charges) or accrual basis (exclude Purchases for future charges). For instance, charges categorized as [ChargeCategory](#chargecategory) "Purchase" and their related [ChargeCategory](#chargecategory) "Tax" charges for a [Commitment-Based Discount](#glossary:commitment-based-discount) might be excluded from an accrual basis cost aggregation of Contracted Cost. This is because the "Usage" and "Tax" charge records provided during the term of the commitment discount already specify the Contracted Cost.
```

Comment:
If we're going to require people to filter rows out in order to use some of our most important columns, we need a prescriptive way for them to apply that filter. If we don't have that, then I would lean towards not enabling such a non-obvious, breaking scenario.

Reply:
I believe this double counting will also happen with the proposed `ActualQuantity` column which shows purchase quantities and actual usage quantities in the same column. So you'd similarly need to filter when aggregating on `ActualQuantity` for either commitment purchases or commitment usage.

I think it's just a consequence of showing two directions of transactions with the same columns.

Reply:
This conversation was resolved by the group, pending a new issue/PR to be raised by Michael.